### PR TITLE
Add a package for Universal Profiling Collector

### PR DESCRIPTION
**Pull request by florianl**
## What does this PR do?

Add a package for Universal Profiling Collector

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/universal_profiling_collector/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    inputs:
+      - title: universal_profiling_collector
+        description: Collect traces from Universal Profiling Agents
+        type: pf-elastic-collector
+        vars:
+          - name: hack
+            type: bool
+            title: hack to workaround a known issue
+            description: |
+              Check out elastic/kibana/issues/155064 for the reason of this hack.
```
Comment (inge4pres) [inline packages/universal_profiling_collector/manifest.yml:28]: AFAIR this is not needed anymore?
Will test it also without the variable and report

File: packages/universal_profiling_collector/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+name: profiler_collector
+title: Universal Profiling Collector
+version: 8.9.0-preview
+categories: ["elastic_stack", "monitoring"]
+description: Fleet-wide, whole-system, continuous profiling with zero instrumentation.
+conditions:
+  kibana.version: ^8.8.0
```
Comment (inge4pres) [inline packages/universal_profiling_collector/manifest.yml:7]: Should this be 8.9.0?
In 8.8 we still have the collector in APM-server, and i am not sure it is meaningful to make them co-exist.

Comment (florianl) [reply packages/universal_profiling_collector/manifest.yml:7]: co-existing is no problem as both packages are independent from each other. using `^8.8.0` here simplifies testing as we can start testing with the 8.8 stack and don't run into issues with 8.9.x-SNAPSHOTs.

**Review by inge4pres — APPROVED** [review]
👍🏼

**Pull request merged**